src/DonationForms/Rules/PhoneIntlInputRule.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Give\DonationForms\Rules;
+
+use Closure;
+use Give\Helpers\IntlTelInput;
+use Give\Vendors\StellarWP\Validation\Contracts\ValidationRule;
+
+/**
+ * @unreleased
+ */
+class PhoneIntlInputRule implements ValidationRule
+{
+    /**
+     * @unreleased
+     */
+    public static function id(): string
+    {
+        return 'intl-input';
+    }
+
+    /**
+     * @unreleased
+     */
+    public static function fromString(string $options = null): ValidationRule
+    {
+        return new self();
+    }
+
+    /**
+     * @unreleased
+     */
+    public function __invoke($value, Closure $fail, string $key, array $values)
+    {
+        $errorMap = IntlTelInput::getErrorMap();
```

Comment:
What kind of error would happen here? Let's add a description to the doc block here

Reply:
Resolve in 1581db8b4be06cc61fc3284aedf3417539c98290